docs/clusterfuzzlite.md

```diff
@@ -11,23 +11,38 @@ permalink: /
 ClusterFuzzLite makes [fuzzing] part of [Continuous Integration (CI)].
 ClusterFuzzLite is based on [ClusterFuzz].
 
-By fuzzing your code, ClusterFuzzLite can make it more secure as fuzzing is
-highly effective technique for finding bugs in software.
-
-ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build].
+Fuzzing is a highly effective technique for finding bugs in software.
+ClusterFuzzLite makes your code more secure by fuzzing changes before they even
+enter the codebase.
+ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build], getting the
```

Comment:
Add "so":
`ClusterFuzzLite supports [GitHub Actions] and [Google Cloud Build], SO getting the
benefits of continuous fuzzing is simple if you can use these CI systems.`